privacy/index.md

```diff
@@ -12,7 +12,7 @@ Certbot is published by the Electronic Frontier Foundation, the leading nonprofi
 <h2>Information Collection</h2>
 Certbot locally stores a variety of information provided by you, including internal housekeeping information created by the ACME protocol, the certificate information  (domain name, public key, time, and specifically selected options), the IP address of the server, a user agent string, which is the operating system the server is running, plus which plugins you're using with Certbot. If you use the "manual" authenticator plugin on your own laptop, the IP address logged by the CA would be the one your ISP gave you, rather than that of the public server.
 
-In addition to the minimal information necessary to create certificates.  Certbot may also transmit additional information to facilitate reliable operation and debugging of ACME client and server software.  By default this includes a User Agent string containing the operating system and version of the machine on which Certbot is run, plus the plugins that the user selected (you can alter or remove this with the --user-agent flag). It also includes contact information such as an email address, if you choose to supply one, so that the CA can notify you about matters such as expiring, unrenewed certificates; security vulnerabilities; or important changes to its policies. 
+In addition to the minimal information necessary to create certificates Certbot may also transmit additional information to facilitate reliable operation and debugging of ACME client and server software.  By default this includes a detailed User Agent string, which may contain a variety of values, such as the operating system and version of the machine on which Certbot is run and the software version, the plugins that the user selected, the command with which you called Certbot, and certain flags on the command line or fields in config files. You can alter or remove this with the –user-agent flag.  We may also ask you questions in the interface, and Certbot may also transmit the answers to the CA. Certbot also may transmit contact information such as an email address, if you choose to supply one, so that the CA can notify you about matters such as expiring, unrenewed certificates; security vulnerabilities; or important changes to its policies.
```

Comment:
> –user-agent flag

This doesn't look right. (It has collapsed the double hyphens into an en dash.)

Perhaps you could use:
>\`--user-agent\`flag